README.md

```diff
@@ -8,13 +8,13 @@
 ### master branch
 The [master](https://github.com/aws/aws-iot-device-sdk-embedded-C/tree/master) branch hosts the continuous development of the AWS IoT Embedded C SDK libraries. Please be aware that the libraries in master branch may have bugs and performance issues. Consider using the released versions of the AWS IoT Embedded C SDK for production ready software.
 
-### v4_beta_deprecated branch
+### v4_beta_deprecated branch (Formerly, named v4_beta)
```

Comment:
A comma is not needed here

Reply:
Wouldn't the non-comma version be `named v4_beta formerly`?

Reply:
No, _"Formerly named v4_beta"_ is fine.